examples/paged_example.py

```diff
@@ -0,0 +1,77 @@
+import torch
+from typing import Tuple
+
+import flag_attn
+
+MAX_SEQ_LEN = 4096
+NUM_BLOCKS = 2000
+
+
+def test_paged_attention(
+    num_seqs: int,
+    num_heads: Tuple[int, int],
+    head_size: int,
+    block_size: int,
+    dtype: torch.dtype,
+    seed: int,
+    device: int,
+):
+    torch.set_default_dtype(dtype)
+    torch.set_default_device(device=device)
+
+    torch.cuda.manual_seed(seed)
+
+    num_query_heads, num_kv_heads = num_heads
+
+    context_lens = torch.randint(1, MAX_SEQ_LEN, [num_seqs], dtype=torch.int32)
+    max_context_len = context_lens.max().item()
+    max_num_blocks_per_seq = (max_context_len + block_size - 1) // block_size
```

Comment:
In autogressive decoding, does the size og block_table gradully grow to 
`(num_seqs, ceil_div(MAX_SEQ_LEN, NUM_BLOCKS))`

Reply:
I think so, this block_table is just for this test case.